src/vxworks.rs

```diff
@@ -11,17 +11,17 @@ use crate::error::{Error, ERRNO_NOT_POSITIVE};
 use core::num::NonZeroU32;
 
 extern "C" {
-    fn randBytes (buf: *mut u8, length: i32) -> i32;
+    fn randBytes(buf: *mut u8, length: i32) -> i32;
```

Comment:
Yes: confirming that the C prototype of randBytes in VxWorks is:

STATUS randBytes
    (
    unsigned char * randBuf,   /* buffer to be written */
    int             numOfBytes /* number of random bytes requested */
    )

where STATUS is a typedef to int, and int is "i32" on all VxWorks targets.

Reply:
What will happen if you'll provide a negative length? In other words, my question is: is it indeed `i32` used for length and not `u32`?

Reply:
>  is it indeed i32 used for length and not u32?
That's a great question, no i32 is not "really" used for the length.

I don't know the the history of this API (i.e. why didn't we use `size_t`), but from what I can tell, under the hood the argument is cast to size_t (so a negative input will turn into a very large number).